deploy/olm-catalog/community-kubevirt-hyperconverged/1.13.0/manifests/kubevirt-hyperconverged-operator.v1.13.0.clusterserviceversion.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     certified: "false"
     console.openshift.io/disable-operand-delete: "true"
     containerImage: quay.io/kubevirt/hyperconverged-cluster-operator:1.13.0-unstable
-    createdAt: "2024-06-20 05:04:59"
+    createdAt: "2024-06-23 16:48:29"
```

Comment:
@nunnatsa 
is it fine we have this change ?
`make build-manifests` done that

Reply:
yes. this change is expected if there additional changes in the file

Reply:
but there arent
unless hco00.crd.yaml in the same folder is considered as well